### ISSUE TITLE
MongoDB projection not working for covered queries

### ISSUE THREAD

**Issue by Sammaye** [yiisoft/yii2#7010]
I have this:
```php
		$query = new Query();
		$query
			->from(Meet::collectionName())
			->where(['user_id' => $user->_id])
			->select(['target_user_id' => true, '_id' => false])
			->limit(5000);

		foreach($query->each() as $k => $v){
			var_dump($v); exit();
			$ninIds[] = $v['target_user_id'];
		}

```

Which I wish to use to make a covered query however, it doesn't work, instead it brings back:
```php
array (size=1)
  '_id' => 
    object(MongoId)[108]
      public '$id' => string '54a311b56803faaf098b4567' (length=24)
```

I believe I know why https://github.com/yiisoft/yii2-mongodb/blob/master/Query.php#L381 because omitting and including of fields is never checked, it is just assumed they are included and since MongoDB must return _id that is what I get.

**Comment by Sammaye** [comment]
tested as working:
```php
	/**
	 * Composes select fields from raw [[select]] value.
	 * @return array select fields.
	 */
	private function composeSelectFields()
	{
		$selectFields = [];
		if (!empty($this->select)) {
			foreach ($this->select as $fieldName => $inc) {
				if(is_string($fieldName)){
					$selectFields[$fieldName] = $inc;
				}else{
					$selectFields[$fieldName] = true;
				}
			}
		}
		return $selectFields;
	}
```

**Comment by klimov-paul** [comment]
Issue resolved by commit 6e50783
